src/script.ts

```diff
@@ -0,0 +1,181 @@
+import { handleTableClick } from "./tableActionButton.ts";
+
+const api =
```

Comment:
it is a good idea to extract constants to a separate file. Although here there is only 1 constant there could be many in bigger apps